tests/core/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ set(TEST_SOURCES
     hakka_json_invalid.cpp
     hakka_json_null.cpp
     hakka_json_object.cpp
+    hakka_json_python_hashing.cpp
```

Comment:
Tests should follow the existing style: please don’t introduce inconsistent patterns. Take a closer look at each test file; you’ll find hashing tests for each type. Please update those tests accordingly